opennms-container/minion/container-fs/confd/templates/org.apache.karaf.management.cfg.tmpl

```diff
@@ -0,0 +1,104 @@
+
```

Comment:
I don't think we've done this yet - but maybe we should add a `# Managed with confd - DO NOT EDIT` style "header" to these files.

Reply:
Added header to the new templates I introduced.